tests/interactive/output.cpp

```diff
@@ -104,17 +104,17 @@ void InteractiveOutputTestCase::TestDllListLoaded()
     for ( size_t n = 0; n < count; ++n )
     {
         const wxDynamicLibraryDetails& details = dlls[n];
-        printf("%-45s", (const char *)details.GetPath().mb_str());
+        wxPrintf("%-45s", (const char *)details.GetPath().mb_str());
```

Comment:
If we use `wxPrintf()`, we should drop `mb_str()` and the cast too, i.e. just pass it `details.GetPath()` directly.

(here and below)